lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/ContextLifeCycle.cs

```diff
@@ -71,10 +71,10 @@ public void Start()
             IContextStart contextStart = new ContextStartImpl(Id);
             
             // TODO: enable
-            //foreach (IObserver<IContextStart> startHandler in _contextStartHandlers)
-            //{
-            //    startHandler.OnNext(contextStart);
-            //}
+            //// foreach (IObserver<IContextStart> startHandler in _contextStartHandlers)
```

Comment:
This style guide says to either use "//" followed by a single space, or use "////" followed by no space (except code indentation). This change uses both "////" and single space, so that looks weird to me.